CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## 2.2.1 - 2023-08-10
```

Comment:
```suggestion

```
Please remove this line, that's what the `## [Unreleased]` heading is for. We'll add that line, with the correct version and date once we cut a release.